### DISCUSSION TITLE
Splunk Slack Notification Alert App deprecated?

### DISCUSSION THREAD

**Discussion by ajaimes98** [splunk/slack-alerts#29] [Q&A]
@ziegfried Hello! 
QQ, is the 'Splunk Slack Notification Alert App' still being supported? or has it been replaced by the ' Splunk Slack App Alert Integration'?

Thanks!

**Comment by bclarkejr** [comment]
Hey @ajaimes98 - We recently updated this alert action to support Slack Apps, so the app should still work. See https://github.com/splunk/slack-alerts/pull/28.

Still waiting on this to get released to Splunk Base, but this app is at least still being supported by @ziegfried.
